packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -807,117 +233,59 @@ class VmServiceWrapper implements VmService {
     }
     return fallbackStream;
   }
-  // Mark: end overrides for the [DdsExtension].
 
-  // Mark: Overrides for [DartIOExtension]. It would help with logical grouping
-  // to make these extension methods, but that makes testing more difficult due
-  // to mocking limitations for extension methods.
-  Future<Version> getDartIOVersion(String isolateId) =>
-      trackFuture('_getDartIOVersion', _vmService.getDartIOVersion(isolateId));
+  // Begin Dart IO extension method wrappers. We wrap these methods so that we
+  // can override them in tests.
```

Comment:
I don't love this but I'm not sure if there is another way to do this given that we need to fake these methods in tests.